base/applications/mspaint/lang/pl-PL.rc

```diff
@@ -32,7 +33,7 @@ BEGIN
         MENUITEM "U&staw jako tło (wyśrodkowane)", IDM_FILEASWALLPAPERCENTERED
         MENUITEM "Ustaw jako tło (rozciągnięte)", IDM_FILEASWALLPAPERSTRETCHED
         MENUITEM SEPARATOR
-        MENUITEM "Ostatnio używany plik", IDM_FILEMOSTRECENTLYUSEDFILE, GRAYED
+        MENUITEM "Niedawno używany plik", IDM_FILEMOSTRECENTLYUSEDFILE, GRAYED
```

Comment:
Just because Vista has such translation doesn't mean we have to replace the better translations that we have. Please keep the current translation here.